docs/development/using_hdk.rst

```diff
@@ -34,4 +34,20 @@ If for some reasons ``Native`` engine is explicitly set using ``modin.config`` o
    If you encounter ``LLVM ERROR: inconsistency in registered CommandLine options`` error when using HDK,
    please refer to the respective section in :doc:`Troubleshooting </getting_started/troubleshooting>` page to avoid the issue.
 
-.. _HDK: https://github.com/intel-ai/hdk
\ No newline at end of file
+
+Running on a GPU
+----------------
+Prerequisites:
+* HDK’s GPU mode is currently supported on Linux and Intel GPU only.
```

Comment:
```suggestion
----------------

Prerequisites:
* HDK's GPU mode is currently supported on Linux and Intel GPU only.
```

Reply:
Leaving a space in between did not render properly for me if it wasn't a synchronization problem

Reply:
It is rendered well https://modin--6326.org.readthedocs.build/en/6326/development/using_hdk.html#running-on-a-gpu.

Reply:
Yeah, I noticed, guess it was a sync issue then